### PR TITLE
Merge numpy.mxnet.io into mxnet official website

### PR DESCRIPTION
**Pull request by ys2843**
## Description ##
Implementation for issue #18566 . 

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:

### Changes ###
- [x] Merge contents from numpy.mxnet.io to main site

## Comments ##
- Preview: http://ec2-34-219-134-42.us-west-2.compute.amazonaws.com/api/python/docs/api/

### CODE REVIEW

**Comment by szha** [comment]
@leezu it would be great if you could take a look at the content as part of reviewing this PR.

**Comment by ys2843** [comment]
@mxnet-label-bot add [Website, pr-awaiting-review]

**Comment by leezu** [comment]
@mxnet-bot run ci [centos-cpu]

File: docs/python_docs/python/tutorials/getting-started/crash-course/1-ndarray.md [hunk 1]
```diff
@@ -15,113 +15,108 @@
 <!--- specific language governing permissions and limitations -->
 <!--- under the License. -->
 
-# Manipulate data with `ndarray`
+# Step 1: Manipulate data with NP on MXNet
 
-We'll start by introducing the `NDArray`, MXNet’s primary tool for storing and transforming data. If you’ve worked with `NumPy` before, you’ll notice that an NDArray is, by design, similar to NumPy’s multi-dimensional array.
+This getting started exercise introduces the `np` package, which is the primary tool for storing and
```
Comment (leezu) [inline docs/python_docs/python/tutorials/getting-started/crash-course/1-ndarray.md:20]: Let's summarize the extent to which `np` is similar and link to a document containing details ` docs/python_docs/python/tutorials/getting-started/deepnumpy/deepnumpy-vs-numpy.md`

File: docs/python_docs/python/tutorials/getting-started/crash-course/1-ndarray.md [hunk 2]
```diff
@@ -15,113 +15,108 @@
...
-```{.python .input  n=1}
-# If you haven't installed MXNet yet, you can uncomment the following line to
-# install the latest stable release
-# !pip install -U mxnet
+To get started, run the following commands to import the `np` package together with the NumPy extensions package `npx`. Together, `np` with `npx` make up the NP on MXNet front end.
 
-from mxnet import nd
+```{.python .input  n=1}
+from mxnet import np, npx
+npx.set_np()  # Activate NumPy-like mode.
```
Comment (leezu) [inline docs/python_docs/python/tutorials/getting-started/crash-course/1-ndarray.md:30]: Linking https://github.com/apache/incubator-mxnet/pull/18631 as the line needs removal after https://github.com/apache/incubator-mxnet/pull/18631 is merged

File: docs/python_docs/python/tutorials/getting-started/crash-course/2-nn.md [hunk 3]
```diff
@@ -15,47 +15,50 @@
...
+x = np.random.uniform(-1,1,(3,4))
 layer(x)
 ```
 
-As can be seen, the layer's input limit of 2 produced a $(3,2)$ shape output from our $(3,4)$ input. Note that we didn't specify the input size of `layer` before (though we can specify it with the argument `in_units=4` here), the system will automatically infer it during the first time we feed in data, create and initialize the weights. So we can access the weight after the first forward pass:
+As can be seen, the layer's input limit of two produced a $(3,2)$ shape output from our $(3,4)$ input. You didn't specify the input size of `layer` before, though you can specify it with the argument `in_units=4` here. The system  automatically infers it during the first time you feed in data, create, and initialize the weights. You can access the weight after the first forward pass, as shown in this example.
 
 ```{.python .input  n=35}
-layer.weight.data()
+# layer.weight.data() # FIXME
```
Comment (leezu) [inline docs/python_docs/python/tutorials/getting-started/crash-course/2-nn.md:56]: FIXME?

Comment (ys2843) [reply docs/python_docs/python/tutorials/getting-started/crash-course/2-nn.md:56]: Sorry for these errors. The contents are copied from Numpy site directly, I think it needs someone who is familiar with the docs to review it.

Comment (leezu) [reply docs/python_docs/python/tutorials/getting-started/crash-course/2-nn.md:56]: For the current case, you can just delete the `#` and `FIXME`. Besides this FIXME and above two comments, I think it's fine to go ahead with merging this PR and have further content improvements in separate PRs

Comment (ys2843) [reply docs/python_docs/python/tutorials/getting-started/crash-course/2-nn.md:56]: Got it, will do. Thank you for reviewing.

Comment (ys2843) [reply docs/python_docs/python/tutorials/getting-started/crash-course/2-nn.md]: Done

File: docs/python_docs/python/tutorials/getting-started/crash-course/1-ndarray.md (hunk 1 above)
Comment (ys2843) [reply docs/python_docs/python/tutorials/getting-started/crash-course/1-ndarray.md]: Done

File: docs/python_docs/python/tutorials/packages/ndarray/index.rst [hunk 4]
```diff
@@ -45,10 +45,16 @@ NDArray
...
 
 .. toctree::
    :hidden:
    :glob:
 
    *
-   sparse/index
\ No newline at end of file
+   sparse/index
+   deepnumpy/index
```
Comment (szha) [inline docs/python_docs/python/tutorials/packages/ndarray/index.rst:60]: this should be at top level, not under ndarray.

Comment (ys2843) [reply docs/python_docs/python/tutorials/packages/ndarray/index.rst:60]: Will move this part to top level

**Pull request merged**